src/Rsk.DuendeIdentityServer.AuditEventSink/Adapters/UnhandledExceptionEventAdapter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Duende.IdentityServer.Events;
+using RSK.Audit;
+
+namespace Rsk.DuendeIdentityServer.AuditEventSink.Adapters
+{
+    public class UnhandledExceptionEventAdapter : IAuditEventArguments
+    {
+        private readonly UnhandledExceptionEvent evt;
+
+        public UnhandledExceptionEventAdapter(UnhandledExceptionEvent evt)
+        {
+            this.evt = evt ?? throw new ArgumentNullException(nameof(evt));
+        }
+
+        public ResourceActor Actor => new ResourceActor(ResourceActor.MachineSubjectType, evt.ActivityId, evt.Details);
```

Comment:
Here, I don't there's any actor. Can we set actor identifier and displayName as null or an empty string maybe? I don't think the event has any properties on it that could determine the actor.